app/uk/gov/hmrc/customs/declarations/information/services/declarationsInformationServices.scala

```diff
@@ -127,7 +123,7 @@ class DeclarationSearchService @Inject()(searchResponseFilterService: SearchResp
       case "cds60010" => backendCDS60010GoodsLocationCodeInvalidResponse
       case "cds60011" => backendCDS60011SubmissionChannelInvalidResponse
       case "cds60012" => backendCDS60012PageNumberInvalidResponse
-      case _          => ErrorInternalServerError
+      case _ => ErrorInternalServerError
```

Comment:
can these have the same indention as before?

Reply:
Not sure how this happened, but will do.